### DISCUSSION TITLE
posts/techorama/

### DISCUSSION THREAD

**Discussion by giscus** [basvandesande/azurecodingarchitect#7] [General]
# posts/techorama/

Tinkering with Azure and other technologies

https://azurecodingarchitect.com/posts/techorama/

**Comment by RikGr** [comment]
Nice recap Bas! Indeed, the social aspect should be one of the key reasons to attend I think. With hindsight I should have reached out more. Good tips for next time :).

**Reply by basvandesande** [reply]
Thanks Rick. With this new knowledge in my backpack, I'll focus more on the social part.

**Comment by RemyDuijkeren** [comment]
Great write up. The social part and sessions are also why I like Techorama a lot. I used to go every year, starting from the first one (Mechelen instead of Antwerpen).

They are really there for the visitors and the partners (how it should be). Like the famous Bart the Smet talks (that give you the feeling that you are still a junior), the gamification of going to the partner boots which helps the partners a lot, the very good food and of course the ice cream truck (they still do that?). The attention to detail by the organizers, which is lost by a lot of 10-in-een-dozijn conferences!

It also started my love for non-technical sessions, I think, but it could also be that fase in my life that technical sessions were not doing it anymore for me.

I went once to the Dutch version, which has the same DNA, but it couldn't live up to the Belgium one.

**Reply by basvandesande** [reply]
Hi Remy, 

for me it was a first timer that I visited Techorama. The line of speakers was incredible and I had a really hard time to pick the sessions I wanted to attend. Bart de Smet is a league of his own, I've seen his presentations a couple of times and every time I feel bedazzled.

Our Belgium colleagues are behind the organization of the event and I was blown away with the presentation and the smoothness of the organization. What really was great was that the morning I arrived at 7.30 I was personally welcomed by them, it makes you feel a bit more special. It turned out that our booth was very popular, I think it had to do something with the great coffee that was being served there.  In discussions with people, the told me that the coffee was their lifesaver.

I think I'll attend the dutch edition in the fall as well. I'm looking forward to seeing  the final speaker and session agenda. The venue this year will be Kinepolis in Utrecht. I think what makes part of the magic, is the location of the venue, in my eyes a multiday event should be far away enough to validate a hotel stay. That makes you that you live in a small bubble for the duration of the event.

I learned this event that I developed a small crush for the non-technical sessions. Being so long in technology it feels like a fresh breeze, changing your perspective on the field you work in.

I hope to meet you in a next event, would be nice to catch up.

Bas

**Reply by RemyDuijkeren** [reply]
"the smoothness of the organization" - this is indeed the overall feel.

Would be nice to catch up, but nowadays I live in Kuala Lumpur so it's a lot harder to be at events in NL ;-).
